paperspace/commands/hyperparameters.py

```diff
@@ -0,0 +1,116 @@
+import terminaltables
+
+from paperspace.commands import common
+
+
+class HyperparametersCommandBase(common.CommandBase):
```

Comment:
does it have to be duplicated?

Reply:
Probably not, but the overall refactor is right behind the corner ;)